src/Apps/Order/Routes/Shipping2/index.tsx

```diff
@@ -72,9 +71,6 @@ const ShippingRouteLayout: FC<Omit<ShippingProps, "dialog">> = ({
 
   const isOffer = order.mode === "OFFER"
 
-  // Automatically selects first shipping quote when they change
-  useSelectFirstShippingQuote()
```

Comment:
Removing this as the logic for selection the first available quote is now available in `ShippingQuotes2` component.